lib/index.js

```diff
@@ -175,16 +190,23 @@ export class Baker extends EventEmitter {
     this.getStaticPath = staticBlock;
```

Comment:
Where else are these used? If these are being used to create static asset paths elsewhere in the proejct then we need to fix those too

Reply:
its being used inside of `prepareCrosswalk` in a project.

example usecase in global-groundwater:
```
prepareCrosswalk(
      /** @type {import('./scripts/types.d').Crosswalk[]} */ data
    ) {
      return data.map((d) => {
        if (d.img) {
          return {
            ...d,
            img: this.getStaticPath(join('assets/img/', d.img)),
            banana: ''
          };
        }
        return { ...d };
      });
    },
  },
```